src/components/reusable/codeView/prismSyntaxStyles.scss

```diff
@@ -0,0 +1,210 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+code[class*="language-"],
+pre[class*="language-"] {
+    text-align: left;
+    white-space: pre;
+    word-spacing: normal;
+    word-break: normal;
+    word-wrap: normal;
+    color: var(--kd-color-text-primary);
+    background-color: var(--kd-color-background-ui-soft);
+    font-family: Roboto Mono, monospace;
```

Comment:
I don't think Roboto Mono is actually getting included anywhere, it's probably falling back to a default monospace font. If we want to actually include Roboto Mono, we might have to add it to Foundation. We also have to self-host it because of GDPR, can't use Google Fonts.